script/release/01.KIE-releases_push-release-branches.sh

```diff
@@ -0,0 +1,84 @@
+BASE_BRANCH=6.4.x
+
+# clone the build-bootstrap that contains the other build scripts
+if [ "$SOURCE" == "community-branch" ]; then
+     
+   # clone droolsjbm-build-bootstrap branch from droolsjbpm
+   git clone git@github.com:droolsjbpm/droolsjbpm-build-bootstrap.git --branch $BASE_BRANCH
```

Comment:
Since this script is already in the droolsjbpm-build-bootstrap git repo, this clone seem unnecessary as the repo already need to exist in order to have access to this script. Or am I missing something?